arm-network/2017-06-01/swagger/loadBalancer.json

```diff
@@ -557,10 +558,11 @@
         },
         "protocol": {
           "type": "string",
-          "description": "The transport protocol for the endpoint. Possible values are: 'Udp' or 'Tcp'",
+          "description": "The transport protocol for the endpoint. Possible values are: 'Udp,' 'Tcp,' or 'All.'",
```

Comment:
What is "All" mean? what if new transport protocols are added to the list, will "All" be still valid?